upup/pkg/fi/nodeup/nodetasks/update_packages.go

```diff
@@ -60,7 +61,8 @@ func (_ *UpdatePackages) RenderLocal(t *local.LocalTarget, a, e, changes *Update
 	glog.Infof("running command %s", args)
 	cmd := exec.Command(args[0], args[1:]...)
 	output, err := cmd.CombinedOutput()
-	if err != nil {
+	// 'yum check-update' exits with 100 if it finds updates; treat it like a success
+	if exitCode := cmd.ProcessState.Sys().(syscall.WaitStatus).ExitStatus(); err != nil && exitCode != 100 {
```

Comment:
I do worry that there may be some errors where cmd.ProcessState is nil (if we fail to even find yum, for example, though admittedly that seems unlikely).  But this LGTM anyway.